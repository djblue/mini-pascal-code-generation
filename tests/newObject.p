program main;

class beep
begin
  var yy, xx : integer;
end

class main 
begin
  var bb : beep;
  function main;
  begin
    bb := new beep;
    bb.xx := 56;
    print bb.xx
  end
end
.
