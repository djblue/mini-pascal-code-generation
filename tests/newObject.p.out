56
24
5
4
3
2
1
1000
100
10
1
